push_dockerhub.sh

```diff
@@ -0,0 +1,3 @@
+docker login -u <username> -p <password>
+# Push the docker image to dockerhub
+docker push rhaoio/prover-node-dev:latest
```

Comment:
For version tag, we can have a meeting to discuss.
Currently maybe just tag it as our playground version like 0.0.14? Just a tag to mention this prover image is for which version of the rest server?